src/main.py

```diff
@@ -0,0 +1,34 @@
+import uvicorn
+from fastapi import FastAPI
+from src.common import config_reader
+from src.commands.command_listener import CommandListener
+
+
+""" uvicorn src.main:app --reload """
+
+app = FastAPI()
+
+command_listener = CommandListener("nikola")
+
+
+@app.get("/")
```

Comment:
I think we should make this /health. In this method we should check if we are still connected to the mqtt server and return json result  { "status" :  "{if all the services are ok then "ok", else "failed"} ( i think we will have only 1 service so this will be the result of the mqtt service )" , "services" : ["mqtt" : "ok (connected) / "failed" (not connected) ]

Reply:
Added as a TODO https://github.com/NHristovski/ServerHelper-ClientApp/pull/1/commits/39775126e7794bb268cfb313ad02b6bb0f081cca.